README.md

```diff
@@ -12,17 +12,23 @@ Use your text editor to write in your browser. Everything you type in the editor
     + [**Sublime Text** extension](https://sublime.wbond.net/packages/GhostText) - [Repo](https://github.com/GhostText/GhostText-for-SublimeText)
     + [**Atom** package](https://github.com/GhostText/GhostText-for-Atom)
     + [**VS Code** extension](https://marketplace.visualstudio.com/items?itemName=tokoph.ghosttext) - [Repo](https://github.com/jtokoph/ghosttext-vscode) (Third party)
-    + [**Vim** script](https://github.com/falstro/ghost-text-vim) (Third party)
-    + [**Vim** plugin](https://github.com/pandysong/ghost-text.vim) (Third party)
-    + [**Neovim** plugin](https://github.com/raghur/vim-ghost) (Third party)
     + [**Emacs** package](https://melpa.org/#/atomic-chrome) - [Repo](https://github.com/alpha22jp/atomic-chrome) (Third party)
     + [**Acme** client](https://github.com/fhs/Ghost) (Third party)
+    + <details>
+        <summary><b>Vim</b>/<b>Neovim</b> (Third party)</summary>
+        <ul>
+          <li><a href="https://github.com/subnut/nvim-ghost.nvim"><b>Neovim</b></a>
+          <li><a href="https://github.com/raghur/vim-ghost"><b>Vim</b> (<tt>+python3</tt>) & <b>Neovim</b> (<tt>pynvim</tt>)</a>
```

Comment:
Is a `+` missing here?
```suggestion
          <li><a href="https://github.com/raghur/vim-ghost"><b>Vim</b> (<tt>+python3</tt>) & <b>Neovim</b> (<tt>+pynvim</tt>)</a>
```

Reply:
No. The things beginning with `+` are compilation flags.
`pynvim` is a python package, installed using `pip` (pip is to python what npm is to node)